src/js/script.js

```diff
@@ -353,6 +415,22 @@
         thisCart.dom.wrapper.classList.toggle(classNames.cart.wrapperActive);
       });
     }
+
+    add(menuProduct) {
```

Comment:
Aby uniknąć powtarzania tego samego kodu z różnych Pull Requestów, twórz kolejne branche z brancha, który jest zależny, np. jeżeli branch 9.4 jest nadbudową brancha 9.3, to stwórz branch 9.4 z brancha 9.3 (git checkout -b 9.4 na branchu 9.3). Następnie merguj/twórz pull requesty w odwrotnej kolejności, np. PR 9.4 -> 9.3 i następny 9.3 -> main